Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Models/LambdaSerializerInfo.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Amazon.Lambda.Annotations.SourceGenerator.Models
+{
+    public class LambdaSerializerInfo
+    {
+        public LambdaSerializerInfo(string serializerName, string serializerJsonContextName)
+        {
+            SerializerName = serializerName;
+            SerializerJsonContextName = serializerJsonContextName;
+        }
+
+        public string SerializerName { get; }
```

Comment:
nit: I like documenting even internal classes, including sample or expected values for strings. As a reviewer not too familiar with JSON+trimming, easier to see here than working backwards from call sites.